test/backend/simple_test.rb

```diff
@@ -72,7 +72,11 @@ def setup
 
   test "simple load_yml: loads data from a YAML file" do
     data = I18n.backend.send(:load_yml, "#{locales_dir}/en.yml")
-    assert_equal({ 'en' => { 'foo' => { 'bar' => 'baz' } } }, data)
+    if ::YAML.respond_to?(:unsafe_load_file)
+      assert_equal({ :en => { :foo => { :bar => 'baz' } } }, data)
```

Comment:
I'd like a check for frozen values here too, please.

Reply:
Sure, added!